src/g_sv_cmds.c

```diff
@@ -618,7 +619,16 @@ __cdecl void G_Say(gentity_t *ent, gentity_t *target, int mode, const char *chat
         return;
 
     // echo the text to the console
-    Com_Printf(CON_CHANNEL_SERVER,"Say %s: %s\n", name, text);
+    const int isPlaying = ent->client->sess.sessionState == SESS_STATE_PLAYING;
+    const int isAttack = ent->client->sess.cs.team == TEAM_BLUE;
+    const int isDefence = ent->client->sess.cs.team == TEAM_RED;
+    const int c1 = isAttack ? 1 : 0;
+    const int c2 = isAttack ? 34 : (isDefence ? 33 : 0);
+    if(com_ansiColor && com_ansiColor->integer) {
+        Com_Printf(CON_CHANNEL_SERVER,"\033[%d;%dm%s%s%s\033[0m: %s\n", c1, c2, isPlaying ? "" : "(Dead)", mode == SAY_ALL ? "" : (isAttack ? "(Attack)" : "(Defence)"), name, text);
+    } else {
+        Com_Printf(CON_CHANNEL_SERVER,"%s%s%s: %s\n", isPlaying ? "" : "(Dead)", mode == SAY_ALL ? "" : (isAttack ? "(Attack)" : "(Defence)"), name, text);
+    }
```

Comment:
Should be optional

Reply:
Made optional in 75af5e23fcdc56ebb621fc5b4135e82ec342c5b7